### PR TITLE
Updated copy for LUC page and made cost breakdown more mobile friendly

### PR DESCRIPTION
**Pull request by danthaman44**
Screen Shot 2020-11-15 at 4 04 31 PM
Screen Shot 2020-11-15 at 4 04 38 PM
Screen Shot 2020-11-15 at 4 04 56 PM

### CODE REVIEW

File: src/locales/cn/translation.json [hunk 1]
```diff
@@ -175,38 +175,43 @@
...
+    "summaryBody2": "Patrick Mock, manager of 46 Mott St Bakery and community advocate, had an idea to light up Chinatown streets to draw customers back to Chinatown. And with the help of <1>Jenny Low</1>, <1>Chung Seto</1> of UDO, <1>Joanne Kwong</1> of Pearl River Mart and <1>Send Chinatown Love</1>, the Light Up Chinatown project was born.",
+    "summaryBody3": "Support the Light Up Chinatown project by donating today. All proceeds will go towards installing permanent light fixtures and traditional lanterns in the neighborhood. Participate in our <1>“Adopt-a-Lantern”</1> initiative with a $45 donation: your personalized lantern will be hung up on Mott Street. Donate $150 or more and you will receive an additional lantern to take home as a keepsake. You will also be invited to the “Light Up Chinatown” ceremony in December celebrating the Winter Solstice.",
+    "summaryBody4": "Help us show that Chinatown is open for business and light the path forward for the neighborhood’s small business-owners by supporting the Light Up Chinatown project today.",
     "campaignHeader": "Light Up Chinatown Fundraising Campaign:",
     "campaignDates": "November 1 - 30, 2020",
```
Comment (aliceluo2048) [inline src/locales/cn/translation.json:183]: Can you update to November 16  - December 15 too for campaign dates

File: src/locales/cn/translation.json [hunk 2]
```diff
@@ -175,38 +175,43 @@
...
     "progressBarHeader": "Total Raised: ",
     "raiseGoal1": " GOAL 1: $23.5k",
-    "raiseGoal2": " GOAL 2: $47k",  
+    "raiseGoal2": " GOAL 2: $47k",
     "ceremony1": "Our big reveal of “Light Up Chinatown” is planned for the second week of December in celebration of the Winter Solstice. We hope you and your family will join us for this festive evening. Together, we can find light in the darkness.",
     "ceremony2": "When you make a lantern donation, you will receive an email once we finalize the date and time.",
     "donationTitle": "Donate to Light Up Chinatown",
     "donationDescription1": "100% of the donations, no matter how big or small, will go towards permanent light fixtures and lanterns throughout Chinatown.",
-    "donationDescription2": "If you donate between $45 and $150 or more, you will be eligible to participate in “Adopt-a-Lantern” where you will be able to receive some perks with your generous donation.",
+    "donationDescription2": "If you donate between more than ${{limit_1}}, you will be eligible to participate in “Adopt-a-Lantern” where you will be able to receive some perks with your generous donation.",
```
Comment (aliceluo2048) [inline src/locales/cn/translation.json:192]:
```suggestion
    "donationDescription2": "If you donate more than ${{limit_1}}, you will be eligible to participate in “Adopt-a-Lantern” where you will be able to receive some perks with your generous donation.",
```

File: src/locales/cn/translation.json [hunk 3]
```diff
@@ -175,38 +175,43 @@
...
+    "faqQuestion7": "What is Send Chinatown Love?",
+    "faqAnswer7": "Send Chinatown Love is a New York-based, entirely volunteer-run organization that helps the Chinatowns across the city get back on their feet by supporting immigrant-owned, small businesses that make up the fabric of these neighborhoods. The goal of Send Chinatown Love is to facilitate community support by creating digital platforms for restaurants and stores in need so that they may be able to sustain themselves now and into the future. Check out our full list of partner merchants <3>here<3>.",
+    "donationDetail1": "1 x 12-inch lantern with your personalization hung on Mott Street",
     "donationDetail2": "Funding permanent light fixtures throughout Chinatown",
-    "donationDetail3": "1 x 12-inch lantern with your personalization hung on Mott Street",
-    "donationDetail4": "1 x 12-inch custom designed lantern by a local artist to take home with you ",
+    "donationDetail3": "1 x 14-inch lantern with your personalization hung on Mott Street",
+    "donationDetail4": "1 x 14-inch custom designed lantern by a local artist to take home with you ",
```
Comment (aliceluo2048) [inline src/locales/cn/translation.json:211]: I think the lantern to take home is 12 inch per Figma design copy

**Review by jacoblb64 — APPROVED** [review]
Looks all good to me! I don't have @aliceluo2048's eagle eyes for copy though, looks like she had some important remarks before we should merge (or fix shortly after)

**Comment by danthaman44** [comment]
@aliceluo2048 Addressed all your feedback

**Pull request merged**